aidants_connect_web/views/espace_responsable.py

```diff
@@ -81,10 +85,43 @@ def aidant(request, organisation_id, aidant_id):
     if aidant.organisation.id != organisation_id:
         raise Http404
 
+    form = RemoveCardFromAidantForm()
+
+    if request.method == "POST":
```

Comment:
Perso, je ferais plutôt l'inverse, ici:

```suggestion
    if request.method == "GET":
        return render(
            request,
            "aidants_connect_web/espace_responsable/aidant.html",
            {"aidant": aidant, "organisation": organisation, "form": form},
        )

    form = RemoveCardFromAidantForm(request.POST)
    # etc.
```

L'interêt, c'est surtout sortir du code le plus rapidement possible sur les chemins de code les plus cours pour améliorer la lisibilité. Comme ça, tu te retrouves pas avec un énorme bloc de code sous le `if`.

Je rajouterais aussi `@require_http_methods(["GET", "POST"])` sur la function pour bien noter qu'il n'y a pas d'autre chemin possible que `GET` et `POST`.

Reply:
Oui tu as raison, return early et évitons ces gros blocs if.
Mais en termes de lisibilité, je crois que je préférerais surtout une méthode `dissociateCard`, je l'ai cherchée ce matin après une semaine loin de ce code. :)

Reply:
Update : c'est fait